build/Dockerfile

```diff
@@ -32,6 +32,5 @@ RUN mkdir -p ${HOME}/.ksctl/cred && \
     mkdir ${HOME}/.ksctl/config/aws && \
     mkdir ${HOME}/.ksctl/config/local
 
-RUN wget -O kubectl https://dl.k8s.io/release/v1.25.0/bin/linux/amd64/kubectl
-
-RUN chmod +x kubectl && mv ./kubectl /usr/local/bin/kubectl
+RUN wget -O kubectl https://dl.k8s.io/release/v1.25.0/bin/linux/amd64/kubectl && \
+    chmod +x kubectl && mv ./kubectl /usr/local/bin/kubectl
```

Comment:
@jonathino2590 we should not make 2 RUN's as it increases the number of Layers in Docker build thus increasing the image size

Reply:
Correct that is the change that is proposed

Reply:
But it will degrade the Docker build time and space @kranurag7